src/v0/destinations/facebook_conversions/utils.js

```diff
@@ -44,7 +44,11 @@ const getCategoryFromEvent = (eventName) => {
   return category;
 };
 
-const populateContentsAndContentIDs = (productPropertiesArray, fallbackQuantity) => {
+const populateContentsAndContentIDs = (
+  productPropertiesArray,
+  fallbackQuantity,
+  fallbackDeliveryCategory,
+) => {
   const contentIds = [];
   const contents = [];
   if (Array.isArray(productPropertiesArray)) {
```

Comment:
@sandeepdsvs  with check of `isObject(productProps)`, Does contentIds & contents gets loaded for `product added to wishlist` ?
https://github.com/rudderlabs/rudder-transformer/blob/2055dabb25e0e67a61dda263ca75f93408f1bf9c/src/v0/destinations/facebook_conversions/utils.js#L129

Reply:
Yes KC, they do get loaded.

Reply:
for `product added to wishlist` `productPropertiesArray` will be of length 1 and `message.properties`, if present, will pass `isObject(productProps)` condition